modules/custom-status/custom-status.php

```diff
@@ -1243,515 +1309,544 @@ function print_configure_view() {
 				<div class="col-wrap">
 				<div class="form-wrap">
 				<h3 class="nav-tab-wrapper">
-					<a href="<?php echo esc_url( $this->get_link() ); ?>" class="nav-tab<?php if ( !isset( $_GET['action'] ) || $_GET['action'] != 'change-options' ) echo ' nav-tab-active'; ?>"><?php _e( 'Add New', 'edit-flow' ); ?></a>
-					<a href="<?php echo esc_url( $this->get_link( array( 'action' => 'change-options' ) ) ); ?>" class="nav-tab<?php if ( isset( $_GET['action'] ) && $_GET['action'] == 'change-options' ) echo ' nav-tab-active'; ?>"><?php _e( 'Options', 'edit-flow' ); ?></a>
+					<a href="<?php echo esc_url( $this->get_link() ); ?>" class="nav-tab
```

Comment:
Same [inline class suggestion](https://github.com/Automattic/Edit-Flow/pull/734/files?diff=split&w=1#r1655521635) for this one.

Reply:
Done